_posts/2016-10-18-october-2016-what-is-graphql

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "October 2016 Meetup - API Driven Development with GraphQL"
+description: ""
+tags: [node.js, javascript, graphql, api, api driven development]
+---
+{% include JB/setup %}
+
+The Seattle Node.js Meetup is pleased to announce our
+[meetup](xxx)
+on **Wednesday, October 26th,** API Driven Development with GraphQL!
+
+As always, you can find all things Seattle Node.js-related at the group website:
+[http://seanode.github.io/](http://seanode.github.io/). And you can reach out to
+us via email at [{{site.author.email}}](mailto:{{site.author.email}}) or Twitter
+at [@{{site.author.twitter}}](https://twitter.com/{{ site.author.twitter }}).
+
+See you soon!
+
+## Agenda
+
+* **7:00pm - 8:00pm**: Learn what GraphQL is and how you can make it work for you!
+
+* **8:30pm - on**: Mingle, mix, chat things Node.js-related and more as we wind down.
+
+<!-- more start -->
+
+## Talks
+
+### API Driven Development with GraphQL
+
+**[Travis Webb](https://github.com/tjwebb)** - CEO of http://www.balderdash.io
+
+How can GraphQL make developing APIs faster and cleaner? Why does it simplify mocking
+and documenting your backend services? And just what in the world is it, exactly? 
+
+In this talk, we'll explore the basics of GraphQL, why it exists in the first place, 
+and how it can make developing your Node applications way more fun. My goal is to 
+leave you inspired to try out your own GraphQL experiments.
+
+Travis Webb is a software developer in Norfolk VA. He is the founder of Langa, a 
+consulting firm that helps companies succeed with Node.js and open-source software,
+creator of Trails.js (http://trailsjs.io), and father of several cats.
+
+## Venue / Sponsor
+
+**[General Assembly](https://generalassemb.ly/seattle)** has graciously agreed to host the event.
+We encourage you to check out one of the awesome downtown restuarants nearby before or 
+following the event!
+General Assembly is located on the third floor at 1218 3rd Avenue, Seattle, WA, 98101 
+
+[Please RSVP with General Assembly for this event!](xxx)
```

Comment:
Yes, they will be giving us a link soon. I just pinged her today asking about it.